README.md

```diff
@@ -1,5 +1,7 @@
 # pylhe: Python LHE interface
 
+<img src="https://raw.githubusercontent.com/scikit-hep/pylhe/master/docs/_static/img/pylhe-logo.png" alt="pylhe logo" width="250"/>
```

Comment:
@lukasheinrich Is there any reason not to use SVG here? Given that we have it? 

![pylhe-logo](https://user-images.githubusercontent.com/5142394/195385360-5a5c86ec-5a75-4217-8acb-7e2a119a09c7.svg)